generator.yml

```diff
@@ -572,7 +573,7 @@ builds:
   - {<< : *arm64_allno,       << : *stable,           << : *llvm_full,       boot: false, << : *llvm_tot}
   - {<< : *arm64_allyes,      << : *stable,           << : *llvm_full,       boot: false, << : *llvm_tot}
   - {<< : *arm64_alpine,      << : *stable,           << : *llvm_full,       boot: true,  << : *llvm_tot}
-  - {<< : *arm64_fedora,      << : *stable,           << : *llvm_full,       boot: true,  << : *llvm_tot}
+  - {<< : *arm64_fedora_bpf,  << : *stable,           << : *llvm_full,       boot: true,  << : *llvm_tot}
```

Comment:
Was this intentional? I think this was resolved with https://git.kernel.org/linus/e96f2d64c812d9c20adea38a9b5e08feaa21fcf5, which landed in 5.18, so I think this particular conversion can be removed because `stable` is just "the latest stable release".

Reply:
Oops, no, that was a read-o. And yes, I totally failed to adjust the other architectures! Let me fix this properly...